### DISCUSSION TITLE
AssociateAction get select id

### DISCUSSION THREAD

**Discussion by warsemann** [filamentphp/filament#6603] [Help] [answered]
Hi, i have an employee table and employee can have multiple device. I use relation manager for associate devices, i want to store history of these entry and i use the after method from AssociateAction

How can i get the id of the asssociate o disassociate device please?

I try $livewire->record but this doesnt work :(

**Comment by danharrin** [comment, marked as answer]
Maybe `$data['recordId']` ?

**Comment by warsemann** [comment]
Yes it's work, and for the disassociate method ? 

I try before method but $data is empty too

**Comment by warsemann** [comment]
Ok for disassociate it s $record->id, thanks !
